### ISSUE TITLE
scaler[ postgresql ] cannot use scaledobject.spec.triggers.passowrd as connection password

### ISSUE THREAD

**Issue by spwangxp** [kedacore/keda#3600]
### Report

when i set scaledobject.spec.triggers.passowrd as connection password, it's not work ❌, the log shows "password" are not set;
when i set scaledobject.spec.triggers.passowrdFromEnv as connection password, it's worked fine ✅;

![image](https://user-images.githubusercontent.com/24298044/186804219-e922945f-d272-4952-b8ae-aa374a554cb4.png)

### Expected Behavior

the scaledobject.spec.triggers.passowrd works as expect

### Actual Behavior

see report

### Steps to Reproduce the Problem

1. see report

### Logs from KEDA operator

```
2022-08-26T02:33:41Z    ERROR   scalehandler    Error getting scalers   {"object": {"apiVersion": "keda.sh/v1alpha1", "kind": "ScaledObject", "namespace": "default", "name": "pg-scaler"}, "error": "error establishing postgreSQL connection: pq: password authentication failed for user \"admin\""}
```

### KEDA Version

2.8.0

### Kubernetes Version

1.23

### Platform

Other

### Scaler Details

postgresql

**Comment by tomkerkhove** [comment]
I don't think we should do this as this would mean there are hardcoded passwords, we should use trigger authentication instead (which should already be supported I think)

**Comment by tomkerkhove** [comment]
this is also not really a bug as `password` is supported, but through trigger authentication:
https://keda.sh/docs/2.8/scalers/postgresql/#authentication-parameters

**Comment by spwangxp** [comment]
https://github.com/kedacore/keda/pull/3601?w=1#issuecomment-1228127320

**Comment by tomkerkhove** [comment]
The docs that you are referring to is purely for TriggerAuthentication and not for trigger metadata.
